pkg/discovery/dtofactory/node_entity_dto_builder.go

```diff
@@ -239,8 +246,16 @@ func (builder *nodeEntityDTOBuilder) getNodeCommoditiesSold(node *api.Node, clus
 	}
 
 	// Cluster commodity.
+	var clusterCommKey string
+	if len(strings.TrimSpace(builder.clusterKeyInjected)) != 0 {
+		clusterCommKey = builder.clusterKeyInjected
+		glog.V(5).Infof("Injecting cluster key for Node %s with key : %s", node.Name, clusterCommKey)
```

Comment:
I think @ading1977 said we don't have V5.  Maybe changing it to V4?

Reply:
Yes.